### ISSUE TITLE
`__emit(load.u.pri)` with pre/post operators

### ISSUE THREAD

**Issue by Y-Less** [pawn-lang/compiler#568]
### Issue description:

Calling a macro that includes `load.u.pri` with a parameter that has `++` or `--` on totally elides the operator.

### Minimal complete verifiable example (MCVE):

```pawn
#define EXAMPLE(%0) __emit(load.u.pri %0)

main()
{
    new c;
    EXAMPLE(c++);
    printf("%d", c); // Should be 1, is 0.
}
```

### Workspace Information:

* Compiler version:
* Command line arguments provided (or sampctl version):
* Operating System:

**Comment by Daniel-Cortez** [comment]
I'll look into it later, but for now I'll have to wait until the other PRs are dealt with, in order to avoid having to rely on other changes that aren't merged in yet, as it already was in #533 and #539.

**Comment by Y-Less** [comment]
Fair enough,  I hadn't realised just how many were outstanding still!  But:

![image](https://user-images.githubusercontent.com/991078/93672293-1c09a580-faaa-11ea-8163-2851d5feafe8.png)

**Comment by Daniel-Cortez** [comment]
@Y-Less Why did you merge them into the `master` branch instead of `dev` though?
